web/src/app/app.module.ts

```diff
@@ -3,14 +3,17 @@ import { NgModule } from '@angular/core';
 
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
+import { AppMaterialModule } from './app.material-module';
+
```

Comment:
I don't think 2 blank lines are needed here

Reply:
okay Eddie. Will remove it.

Reply:
Thank you 👍